vault/resource_pki_secret_backend_config_cluster.go

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/util"
+)
+
+var pkiClusterFields = []string{
+	consts.FieldPath,
+	consts.FieldAIAPath,
+}
+
+func pkiSecretBackendConfigClusterResource() *schema.Resource {
+	return &schema.Resource{
+		Create: pkiSecretBackendConfigClusterCreateUpdate,
+		Read:   provider.ReadWrapper(pkiSecretBackendConfigClusterRead),
+		Update: pkiSecretBackendConfigClusterCreateUpdate,
+		Delete: pkiSecretBackendConfigClusterDelete,
```

Comment:
Could we use the *Context fields instead. See CreateContext for example: https://github.com/hashicorp/terraform-plugin-sdk/blob/51231d6a243efea783559c73395aa0794d2f9859/helper/schema/resource.go#L279

Reply:
Sure. The change has been made.